autoload/lsp.vim

```diff
@@ -150,6 +150,7 @@ function! s:register_events() abort
         autocmd BufWinLeave * call s:on_text_document_did_close()
         autocmd BufWipeout * call s:on_buf_wipeout(bufnr('<afile>'))
         autocmd InsertLeave * call s:on_text_document_did_change()
+        autocmd TextChanged * call s:on_text_document_did_change()
```

Comment:
It would be good to also have `TextChangedP`, but we need to make sure it exists using `exists('##TextChangedP')`

https://github.com/vim/vim/issues/1691
https://github.com/vim/vim/pull/2372